tools/firely-server-import.rst

```diff
@@ -72,6 +72,7 @@ If you want to specify input parameters in the file, you can use the snippet bel
     "updateExistingResources": true,
     "databaseType": "SQL",
     "haltOnError": false,
+    "convertAbsoluteUrlsToRelative":[]
```

Comment:
Can you also add a short description of the feature to the release notes (line 284)?